templates/civicrm-entity.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * Default template for a CiviCRM template.
+ */
```

Comment:
Perhaps some template naming suggestions documented here for how to make a template for one entity type

Reply:
I concur! I had to look at the code in `hook_theme_suggestions_HOOK_alter()` to figure out how to name the template file. Having it documented somewhere would be good.

@arshad

Reply:
@dsnopek Done. I've updated the comment in `civicrm-entity.html.twig`.

Reply:
Looks great, thanks!